osu.Game/Screens/OnlinePlay/Multiplayer/CreateMultiplayerMatchButton.cs

```diff
@@ -22,6 +22,7 @@ public class CreateMultiplayerMatchButton : CreateRoomButton
         [BackgroundDependencyLoader]
         private void load()
         {
+            SpriteText.Font = SpriteText.Font.With(size: 14);
```

Comment:
This is specified in both inheritors of `CreateRoomButton`. Possible to hoist this up there.